hudi-utilities/src/test/java/org/apache/hudi/utilities/streamer/TestStreamSyncUnitTests.java

```diff
@@ -160,6 +165,45 @@ void testGetCheckpointToResume(HoodieStreamer.Config cfg, HoodieCommitMetadata c
     assertEquals(expectedResumeCheckpoint,resumeCheckpoint);
   }
 
+  @ParameterizedTest
+  @MethodSource("getMultiTableStreamerCases")
+  void testCloneConfigsFromMultiTableStreamer(HoodieMultiTableStreamer.Config cfg) throws IOException {
+    Configuration configuration = new Configuration();
+    JavaSparkContext jssc = mock(JavaSparkContext.class);
+
+    when(jssc.hadoopConfiguration()).thenReturn(configuration);
+
+    HoodieMultiTableStreamer multiTableStreamer = new HoodieMultiTableStreamer(cfg, jssc);
+    List<TableExecutionContext> tableExecutionContextList = multiTableStreamer.getTableExecutionContexts();
+    tableExecutionContextList.forEach(it -> {
+      // make sure that if set global properties then each child streamer can get also
+      assertTrue(it.getConfig().configs.containsAll(cfg.configs));
```

Comment:
Can we also add a check for `cfg.configs` to be non empty?

Reply:
@lokeshj1703 
We should allow to have empty configs.
[Test case 1](https://github.com/apache/hudi/pull/11149/files#diff-d5d9e34abfadbfc00ca3aa1642dbfde1a59ecaf2976ce7a5be418e9ae0a1a17fR192) address that.

Reply:
I just wanted to ensure we have a test which covers the case where the configs are non empty. I see that is already covered based on the test cases.